surveys/promise-rejections/survey.md

```diff
@@ -0,0 +1,68 @@
+# Node.js Promise reject use case survey
+
+## Are you currently using Promises, async/await, a mix, or neither?
+
+Assume using as creating your own `new Promise` and `async function`
+
+ - [ ] Promises
+ - [ ] async/await
+ - [ ] Both
+ - [ ] Neither / not writing Promise-based code
+
+## Are you currently using `.catch()`, `try await catch`, a mix, or neither to handle rejections?
```

Comment:
"`try await catch`" isn't an idiom i've heard to describe what you're describing here; i think it needs elaboration.